packages/lesswrong/lib/collections/revisions/schema.ts

```diff
@@ -294,6 +294,35 @@ const schema: SchemaType<"Revisions"> = {
       resolver: (multiDocumentField) => multiDocumentField('*'),
     })
   }),
+  summary: resolverOnlyField({
+    type: "MultiDocument",
+    graphQLtype: "MultiDocument",
+    canRead: ['guests'],
+    resolver: async (revision: DbRevision, args: void, context: ResolverContext) => {
+      const { currentUser, MultiDocuments } = context;
+      if (revision.collectionName !== "MultiDocuments") {
+        return null;
+      }
+      if (!revision.documentId) {
+        return null;
+      }
+      const lens = await context.loaders.MultiDocuments.load(revision.documentId);
+      if (lens.fieldName !== "summary" || lens.collectionName !== "Tags") {
+        return null;
+      }
+      return await accessFilterSingle(currentUser, MultiDocuments, lens, context);
+    },
+    sqlResolver: ({ field, join }) => join({
+      table: 'MultiDocuments',
+      on: {
+        _id: field('documentId'),
+        collectionName: "'Tags'",
```

Comment:
MultiDocuments (lenses) can also have summaries - I haven't checked how we're handling this elsewhere (if we are, at all)

Reply:
Ah, right. That wasn't handled before, but I have now added handling and tested it.

Reply:
How is it handled?  This still looks like it's limited to retuning summaries for top-level tags.  (I think it can be fixed by just removing the `collectionName` as part of the `on`, since the join on `documentId` and `fieldName` ought to be sufficient to return the correct records)

Reply:
Ah, it looks like I addressed it in the regular resolver but not in the `sqlResolver`, and it passed functional testing because the `sqlResolver` isn't actually used. I'm just going to delete the whole thing; I think having a `sqlResolver` here isn't actually a good idea.

Reply:
Ok, sure